### DISCUSSION TITLE
UI remains active when web browser window is not in focus

### DISCUSSION THREAD

**Discussion by digitalsushi** [rern/rAudio#1431] [Feature Requests]
It could easily be this is forbidden by browser policy, but I'll ask.

On a Mac (13.1), on both Chrome 116.0.5845.140 and Firefox 109.0.1, when I have rAudio running in a browser, the song titles and VU meter stop updating. I believe it happens over a websocket, which I am not familiar with debugging.

I'd like it if it remained active, since I have multiple monitors and it's a nice fixture to see the active song even if the browser is not technically in focus - I can still see it.

Thank you. I am expecting there is a technical blocker to this and happy to hear any answer.

**Comment by feebb** [comment]
I'm on Windows, but I experience the same issue. Whenever the browser loses focus, the rAudio page stops updating. I also assumed it was a browser and not a rAudio issue, but it would be great if the page would update continuously. 

Thanks very much!
